src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,82 @@
+package duke;
+
+import java.time.LocalDate;
+
+import duke.command.*;
```

Comment:
Should wildcard imports be used? I think you can disable wildcard imports in the editor settings in the IDE.